change/@azure-msal-common-2020-08-11-16-45-31-fix-id-token-claims-type.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Change AuthenticationResult.idTokenClaims type from StringDict to generic Object type to allow for non-string valued ID Token Claims. Also updates exp claim type in IdTokenClaims type from string to number (#2105)",
```

Comment:
nit: In the interest of easily digestible change files we should aim to keep these comments pretty small. People can then open the relevant PR for more detail. Something like "Update typing of IdTokenClaims (#2105)" would be fine here.

Reply:
+1

Reply:
Cool, I'll keep this in mind for the future as well!